autoscale_cloudroast/test_repo/autoscale/fixtures.py

```diff
@@ -134,6 +135,17 @@ def _set_up_clients():
 autoscale_client, server_client, lbaas_client, rcv3_client = _set_up_clients()
 
 
+def skip_if_mimic_is(boolean):
```

Comment:
Could you rename the `boolean` to `skip` or something else that describes its intent?